src/flag_gems/__init__.py

```diff
@@ -47,6 +47,7 @@ def enable(lib=aten_lib):
     lib.impl("ge.Scalar", ge_scalar, "CUDA")
     lib.impl("gelu", gelu, "AutogradCUDA")
     lib.impl("native_group_norm", group_norm, "AutogradCUDA")
+    lib.impl("_weight_norm", weight_norm, "CUDA")
```

Comment:
Let's leave _weight_norm to Pytorch and replace _weight_norm_interface only, that'll give us a lot more convenience.

Reply:
_weight_norm_interface also requires a backward function. Please do add it.